omnij-net-api/src/main/java/foundation/omni/netapi/omniwallet/OmniwalletAbstractClient.java

```diff
@@ -171,16 +171,7 @@ protected AddressBalanceEntry balanceMapper(AddressVerifyInfo item, PropertyType
         Address address = item.getAddress();
         OmniValue balance = toOmniValue(item.getBalance(), propertyType);
         OmniValue reserved = toOmniValue(item.getReservedBalance(), propertyType);
-        boolean isFrozen = item.isFrozen();
-
-        OmniValue frozen;
-
-        if (isFrozen) {
-            frozen = balance;
-            balance = OmniValue.of(0, propertyType);
-        } else {
-            frozen = OmniValue.of(0, propertyType);
-        }
+        OmniValue frozen = item.isFrozen() ? balance : OmniValue.of(0, propertyType);
```

Comment:
@achamely This is how we convert from the Omniwallet JSON (`isFrozen` is the Java-style name for the `frozen` boolean) to the `BalanceEntry` structure that is a POJO (plain-old Java object) representing the JSON returned by the Omni Core `omni_getallbalancesforid` RPC.

Reply:
Fore reference, here's a link to the [AddressVerifyInfo](https://github.com/OmniLayer/OmniJ/blob/master/omnij-net-api/src/main/java/foundation/omni/netapi/omniwallet/json/AddressVerifyInfo.java) POJO.